### ISSUE TITLE
OpenAPI schema differs to response, when field using alias

### ISSUE THREAD

**Issue by playpauseandstop** [vitalik/django-ninja#85]
Hi,

I'm a bit confused on how `pydantic` in conjunction with `django-ninja` handle fields with aliases.

Saying, I have the `Address` schema declared as,

```py
from ninja import Schema
from pydantic import ConstrainedStr, UUID4


class NonEmptyStr(ConstrainedStr):
    min_length = 1


class Address(Schema):
    uid: UUID4

    country_code: NonEmptyStr = Field(alias="country")
    state_code: NonEmptyStr = Field(alias="region")

    postal_code: NonEmptyStr
    city: NonEmptyStr = Field(alias="locality")

    address_line1: NonEmptyStr
    address_line2: Optional[NonEmptyStr] = None

    full_address: NonEmptyStr = Field(alias="street_address")
```

The API then respond with **intentional** dataset such as,

```json
{
  "address_line1": "1327 El Prado Ave",
  "address_line2": null,
  "city": "Torrance",
  "country_code": "US",
  "full_address": "1327 El Prado Ave",
  "postal_code": "90501",
  "state_code": "CA",
  "uid": "601ee965-fa89-48ce-b9f3-38260d52e1b6"
}
```

But OpenAPI Schema uses **aliases** instead of field names,

![Screen Shot 2021-02-25 at 20 53 59](https://user-images.githubusercontent.com/21325/109210142-b9c97500-77ac-11eb-8ec1-e50b59c1d5f5.png)

Which means, while API backed by `django-ninja` returns expected results (`city` instead of `locality`), the OpenAPI schema backed by `pydantic` contains unexpected values (`locality` instead of `city`).

Maybe this somehow can be fixed via `Schema.Config`, but I failed to find anything in [documentation](https://pydantic-docs.helpmanual.io/usage/model_config/).

Or maybe `django-ninja` is able to somehow fix `Address.schema()` generator to include expected field names in both responses (as it did already) and OpenAPI schema.

Thanks and please let me know, if you need any other data to reproduce the issue.

- Python version: **3.9.2**
- Django version:  **3.1.7**
- django-ninja version: **0.10.2**
- pydantic version: **1.7.3**

**Comment by vitalik** [comment]
Please test with latest `development` branch

https://github.com/vitalik/django-ninja/tree/development

I'll do few more tests, and if all fine - it will land on v0.11

**Comment by playpauseandstop** [comment]
Hi, `development` branch seems working for me.

Thanks for quick response & fix! Will wait on `v0.11` then

**Comment by playpauseandstop** [comment]
Can confirm it works well in `v0.11`

Thanks for fix!
